src/TeemIpDiscoveryCollectionPlan.class.inc.php

```diff
@@ -102,7 +102,7 @@ public function Init(): void
 
 			// Get Discovery application UUID
 			$aOtherPlaceholders = Utils::GetConfigurationValue('json_placeholders', []);
-			if (array_key_exists('discovery_application_uuid', $aOtherPlaceholders)) {
+			if (array_key_exists('discovery_application_uuid', $aOtherPlaceholders) && !empty($aOtherPlaceholders['discovery_application_uuid'])) {
```

Comment:
Actually, the key always exists as it is present in the params.distrib.xml so only the empty check is enough.

Reply:
@odain-cbd What do you think? Change this if to this?
```php
if (!empty($aOtherPlaceholders['discovery_application_uuid'])) {
```

Reply:
Well, we could. At the same time, it doesn't hurt to keep extra protection.